sdk/openai/Azure.AI.OpenAI/tests/OpenAIInferenceTests.cs

```diff
@@ -26,6 +26,20 @@ public void InstanceTest()
             Assert.That(tokenClient, Is.InstanceOf<OpenAIClient>());
         }
 
+        [Test]
```

Comment:
You should be able to record this test.  Add a sanitizer for the token and look for it env variable.  Add `PlaybackOnly` attribute to the test so that it won't run live.